File: ejercicios parte 1/1)programas simples/6pitagoras/pitagoras.cpp
#include <iostream>
#include <math.h>
using namespace std;
int main(){
    int A,B;
    float C;

    cout<<"Ingrese el cateto A: ";
    cin>>A;
    cout<<"Ingrese el cateto B: ";
    cin>>B;
    C= sqrt(A*A + B*B);
    cout<<"La hipotenusa de este triangulo es:  "<<C<<endl;

    return 0;
}

File: ejercicios parte 1/1)programas simples/6pitagoras/pitagoras.java
import java.util.Scanner;

public class pitagoras {

    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int A, B;
            double H;
            System.out.print("Ingrese el catedo A: ");
            A=scan.nextInt();
            System.out.print("Ingrese el catedo B: ");
            B=scan.nextInt();
            H = Math.sqrt(A*A + B*B);
            System.out.println("La hipotenusa es: "+H);
        }
    }   
}